README.adoc

```diff
@@ -440,6 +442,22 @@ class Transaction < ApplicationRecord
 end
 ----
 
+=== Positional arguments [[positional-arguments]]
+
+Prefer using positional arguments instead of keyword arguments when defining enums.
```

Comment:
I’m on the fence if this should be in the guide.
I can see the Rails team trying to get rid of the code that detects the enum attribute name.

I can also see potential mistakes like `enum status: {a: 1}, priority: {high:1,…}`. But is it common?

But how does that stylistically differ for the rest of us? Why do people need to care? If it’s older Rails - use any of the teo. If it’s new, only the positional would work.

Reply:
You're right - since deprecation notice captures the old syntax, seems like we don't need to have a dedicated cop for this at all, I'll close both PRs 😬 

thanks for the feedback!